docs/audit_report/src/01_generic_changes.rst

```diff
@@ -1,6 +1,59 @@
 Changes Overview
 ================
 
-.. todo::
+Since the previously audited version (|botan_git_base_ref|), Botan
+|botan_version| brings some extensions and fixes. The most relevant changes are outlined below.
 
-   Write a high-level overview of the changes that made it into Botan 3.2.0
+Transport Layer Security
+------------------------
+
+The TLS 1.3 implementation is now capable of establishing secure connections
+using post-quantum secure KEMs. Botan |botan_version| supports hybrid key
+exchanges with the following algorithm combinations:
+
++--------------------------------+--------------------------------+
+| PQ KEM                         | Classical Key Exchange         |
++--------------------------------+--------------------------------+
+| | Kyber R3 512                 | | X25519                       |
+| | Kyber R3 512                 | | NIST P-256                   |
+| | Kyber R3 768                 | | X25519                       |
+| | Kyber R3 768                 | | NIST P-384                   |
+| | Kyber R3 1024                | | NIST P-1024                  |
++--------------------------------+--------------------------------+
+
+Additionally, handshakes with just Kyber as a PQ KEM and without any classical
```

Comment:
![image](https://github.com/sehlen-bsi/botan-docs/assets/126883116/eb1f12c8-e349-4f97-9d0f-6c3a8aa6f59b)

In comparison to other tables this one looks somehow weird. Maybe we want to add row separators.

Reply:
Yeah. Good idea. Done.